index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/styles.css" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous" />
+  <title>Rinse - Home</title>
+</head>
+
+<body>
+  <header>
+    <ul class="top-menu">
+      <li>f</li>
+      <li><span class="bi-twitter"></span></li>
+      <li>en</li>
+      <li>My Page</li>
+      <li>Log out</li>
+    </ul>
+    <nav class="container-fluid">
+      <div class="logo"></div>
+      <button type="button" class="menu-toggle" aria-expanded="false" aria-controls="list-menu" type="button">
+        <span class="sr-only">Menu</span>
+      </button>
+      <ul class="list-menu" id="list-menu" data-visible="false">
+        <li class="menu-item"><a href="about.html">About Us</a></li>
+        <li class="menu-item"><a href="#">Locations</a></li>
+        <li class="menu-item"><a href="#">How to Book</a></li>
+        <li class="menu-item"><a href="#">Pricing</a></li>
+        <li class="menu-item"><a href="#">Cleaning Tips</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section class="container-fluid main-content">
+    <div id="section-1">
+      <h4>"Cleanliness is next to Godliness"</h4>
+      <h1 class="title">HIRE TRUSTED HOUSEKEEPERS READY TO CLEAN</h1>
+      <p>
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consequatur,
+        dolores ipsa quo sunt omnis nemo autem corrupti minima sapiente
+        veritatis rem culpa consectetur cupiditate vitae amet aliquam non
+        necessitatibus iste.
+      </p>
+      <h2 class="c-hours">CLEANING HOURS - 9:00 AM TO 5:00 PM (MON - FRI)</h2>
+      <p class="c-msg">We have the best offers available on the market</p>
+    </div>
+  </section>
```

Comment:
Kindly fix the spacing around elements and introduce some space at the top of this section to resemble the given design in the mobile version 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/190372925-a1f8ed91-e2f4-4312-9f99-bd58244c2408.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190372977-97ac020b-4708-4719-90b2-62c7b8855bbb.png)